sdk/identity/identity/src/credentials/chainedTokenCredential.ts

```diff
@@ -85,8 +87,8 @@ export class ChainedTokenCredential implements TokenCredential {
         "ChainedTokenCredential authentication failed."
       );
       span.setStatus({
-        code: SpanStatusCode.ERROR,
-        message: err.message,
+        status: "error",
```

Comment:
Same here, since you're just logging and throwing, `withSpan` is your friend here